comment_iowriter.go

```diff
@@ -62,6 +64,8 @@ func (mc *UnifiedCommentWriter) Post(_ context.Context, c *Comment) error {
 	return err
 }
 
+func (*UnifiedCommentWriter) ShouldPrependGitRelDir() bool { return false }
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported method UnifiedCommentWriter.ShouldPrependGitRelDir should have comment or be unexported
<!-- __reviewdog__:ChA5Y2VmYjk1YzMxNmVmYThkEg9nb2xpbnQtY2lyY2xlY2k= -->